invenio_rdm_records/requests/community_inclusion.py

```diff
@@ -65,6 +65,10 @@ def execute(self, identity, uow):
         default = not record.parent.communities
         record.parent.communities.add(community, request=self.request, default=default)
 
+        if community.parent:
```

Comment:
Small note: make sure that when this is released, you have the correct `min` version of the modules that provide the `.parent` in the setup :)

Reply:
Maybe for "top" backwards compat we could change to:

```suggestion
        if getattr(community, "parent", None):
```

Maybe then we won't need any bumps + release chain.